releases/release-1.17/README.md

```diff
@@ -0,0 +1,196 @@
+* All Enhancements must have a KEP that is in `implementable` state by Enhancements Freeze. If the enhancement does not have a KEP in `implementable` state by Enhancements Freeze it will be removed from the milestone and will require an [exception](/releases/EXCEPTIONS.md).
+* The Emeritus Adviser will be responsible for advising the Release Team Lead and Lead Shadows, ensuring shadow selection happens, and that release shadows are getting the most out of their participation in the release process.
+* The itemized list of code changes for the release is being migrated from the release changelog `CHANGELOG-x.y.md` (where `x` and `y` are major and minor versions of Kubernetes, respectively) to [https://relnotes.k8s.io](https://relnotes.k8s.io)
+* We will continue to implement guidelines and clean up around CI Signal.
+
+## Timeline
+| **What** | **Who** | **Sept** | **Oct** | **Nov** | **Dec** | **WEEK** | **CI SIGNAL** |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| Team Leads finalized | Lead | Wed 18 | | | | week 0 | |
+| Start of Release Cycle | Lead | Mon 23 | | | | week 1 | [master-blocking] |
+| Start Enhancements Tracking | Enhancements Lead | Tue 24  | | | |  | |
+| Schedule finalized | Lead | Wed 25 | | | | | |
+| Team Shadows finalized | Lead | Wed 25 | | | | | |
+| 1.17.0-alpha.1 released | Branch Manager | | Tue 1 | | | week 2| |
+| Start Release Notes Draft | Release Notes Lead | | Tue 8 | | | week 3 | |
+| **Begin [Enhancements Freeze]** (EOD PST) | Enhancements Lead | | Tue 15 | | | week 4 | [master-blocking], [master-informing] |
+| 1.17.0-alpha.2 released | Branch Manager | | Tue 15 | | | | |
+| 1.17.0-alpha.3 released | Branch Manager | | Tue 22 | | | week 5 | |
+| release-1.13 jobs removed | Branch Manager | | Tue 29 | | | week 6 | |
+| release-1.17 branch created | Branch Manager | | Tue 29 | | | | |
+| 1.17.0-beta.0 released | Branch Manager | | Tue 29 | | | | |
+| release-1.17 jobs created| Branch Manager | | Tue 29 | | | | |
+| **Begin [Burndown]** (MWF meetings) | Lead | | | Mon 4 | | week 7 | [1.17-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]** | Lead | | | Mon 4 | | | |
+| Brace Yourselves, Code Freeze is Coming | Comms / Bug Triage| | | Mon 4 | | | |
+| 1.17.0-beta.1 released | Branch Manager | | | Tue 5| | | |
+| Docs deadline - Open placeholder PRs | Docs Lead | | | Fri 8 | | | |
+| **Begin [Code Freeze]** (EOD PST) | Branch Manager | | | Thu 14 | | week 8 | |
+| KubeCon Begins | Community | | | Mon 18 | | week 9 | |
+| Burndown Meetings daily | Lead | | | Tues 19 | | week 9 | |
+| Docs deadline - PRs ready for review | Docs Lead | | | Tues 19 | | |
+| 1.17.0-beta.2 released | Branch Manager | | |  Wed 20| |
+| Docs complete - All PRs reviewed and ready to merge | Docs Lead | | | Mon 25 | | week 10 | |
+| **Begin [Code Thaw]** (EOD PST) | Branch Manager | | | Tue 26 | | | [1.17-blocking] |
+| 1.17.0-rc.1 released | Branch Manager | | | Tue 26 | | |
```

Comment:
I would suggest putting the rc1 before code thaw, even though they're on the same day (if rc1 slips, we'd likely delay thawing)